packages/fluentui/react-northstar/src/components/Menu/MenuDivider.tsx

```diff
@@ -27,41 +27,72 @@ export interface MenuDividerProps extends UIComponentProps, ChildrenComponentPro
 
 export const menuDividerClassName = 'ui-menu__divider';
 
-class MenuDivider extends UIComponent<WithAsProp<MenuDividerProps>> {
-  static displayName = 'MenuDivider';
+export type MenuDividerStylesProps = Required<Pick<MenuDividerProps, 'vertical' | 'inSubmenu' | 'primary'>> & {
+  hasContent: boolean;
+};
 
-  static create: ShorthandFactory<MenuDividerProps>;
+export const MenuDivider: React.FC<WithAsProp<MenuDividerProps>> &
+  FluentComponentStaticProps<MenuDividerProps> = props => {
+  const context: ProviderContextPrepared = React.useContext(ThemeContext);
+  const { setStart, setEnd } = useTelemetry(MenuDivider.displayName, context.telemetry);
+  setStart();
+  const { children, content, vertical, inSubmenu, primary, className, design, styles, variables } = props;
+  const ElementType = getElementType(props);
+  const unhandledProps = useUnhandledProps(MenuDivider.handledProps, props);
 
-  static deprecated_className = menuDividerClassName;
+  const getA11yProps = useAccessibility<never>(props.accessibility, {
```

Comment:
```suggestion
  const getA11yProps = useAccessibility(props.accessibility, {
```
It's not important, type should be inferred from `props.accessibility`
In the same time for `useStyles` it's important as we can't infer type there